cloud/compose.go

```diff
@@ -143,3 +143,34 @@ func (c Client) ComposeWait(id string, timeout, interval time.Duration) (aborted
 		}
 	}
 }
+
+// TODO Function to just return the whole pile?
```

Comment:
Not sure if we need to keep this comment, if we don't need the function up until now